apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/source/FissClaimTransformerTest.java

```diff
@@ -412,6 +418,48 @@ public void benezPayer() {
         claim.getPayers(), transformed.getPayers(), PreAdjFissPayer::getPriority);
   }
 
+  @Test
+  public void auditTrail() {
```

Comment:
This test name and the others below could be a bit more descriptive. (And documented)

Reply:
Will do.

Reply:
Done.